cbf/v1/cbf-v1.0.0.yaml

```diff
@@ -0,0 +1,591 @@
+          content:
+            application/json:
+              schema:
+                $ref: ../../models/ErrorResponse.yaml
+              examples:
+                Invalid GET Request:
+                  value:
+                    httpMethod: GET
+                    requestUri: 'https://dcsa.org/dcsa/cbf/v1/booking-forcasts'
+                    statusCode: 400
+                    statusCodeText: Bad Request
+                    errorMessage: Either carrierServiceCode or universalServiceReference must be provided.
+                    providerCorrelationID: 4426d965-0dd8-4005-8c63-dc68b01c4962
+                    errorDateTime: '2019-11-12T07:41:00+08:30'
+                    errors: []
+        '404':
+          description: Booking forecast not foound.
+          content:
+            application/json:
+              schema:
+                $ref: ../../models/ErrorResponse.yaml
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: ../../models/ErrorResponse.yaml
+      operationId: get-booking-forecast-by-reference
+      description: |-
+        Fetch the booking forecast by either the `carrierServiceCode` or the `universalServiceReference`.\
+        Either one is **REQUIRED** but not both when fetching the booking forcast.
+      parameters:
+        - schema:
+            type: string
+            pattern: ^\S+(\s+\S+)*$
+            example: FE1
+            maxLength: 11
+          in: query
+          name: carrierServiceCode
+          description: |-
+            The carrier-specific code of the service for which the schedule details are published.\
+            **REQUIRED** if `universalServiceReference` is not provided.
+        - schema:
+            type: string
+            pattern: '^SR\d{5}[A-Z]$'
+            maxLength: 8
+            example: SR12345A
+          in: query
+          name: universalServiceReference
+          description: |-
+            A global unique service reference, as per DCSA standard, agreed by VSA partners for the service. \
+            **REQUIRED** if `carrierServiceCode` is not provided.
+        - schema:
+            type: string
+          in: query
+          name: vesselIMONumber
+          description: 'The unique reference for a registered Vessel. The reference is the International Maritime Organisation (IMO) number, also sometimes known as the Lloyd''s register code, which does not change during the lifetime of the vessel'
+          required: true
+        - schema:
+            type: string
+            pattern: ^\S+(\s+\S+)*$
+            example: 2201N
+            maxLength: 50
+          in: query
+          name: carrierExportVoyageNumber
+          description: The carrier-specific identifier of the export voyage.
+        - schema:
+            type: string
+            pattern: '^\d{2}[0-9A-Z]{2}[NEWS]$'
+            example: 2103S
+            maxLength: 50
```

Comment:
Wrong maxLength - should be 5
Also "R" (Roundtrip) seems to be missing in regEx

Reply:
seems like you made a mistake here @HenrikHL , because I copied it from there https://app.swaggerhub.com/apis/dcsaorg/DCSA_OVS/3.0.0-Beta-2#/Operational%20Vessel%20Schedules/get_v3_service_schedules

Reply:
But the "R" has been decided after I wrote the OVS specs...

Reply:
> But the "R" has been decided after I wrote the OVS specs...

This is what the PO said 
> we didnt decide, but will have to add at the end because CMA won't get rid of it.  so add it.